app/src/main/java/com/esri/arcgisruntime/opensourceapps/datacollection/viewmodels/IdentifyResultViewModel.kt

```diff
@@ -71,16 +81,23 @@ class IdentifyResultViewModel : ViewModel() {
     /**
      * Resets the result of the identify operation
      */
-    fun resetIdentifyResult() {
+    fun resetIdentifyLayerResult() {
         _identifyLayerResult.value = null
-        _identifiedPopup.value = null
     }
 
     /**
      * Called when user taps on the identify result bottom sheet. Kicks off PopupAttributeListFragment
      */
     fun showPopupAttributeList() {
-        _showPopupAttributeListEvent.raiseEvent()
+        _showPopupFragmentEvent.raiseEvent()
+    }
+
+    /**
+     * Called when user taps on the identified popup attribute list header in the expanded bottom sheet
+     * Kicks off IdentifyResultFragment
+     */
+    fun showIdentifiedPopupAttributes() {
+        _showIdentifyResultFragmentEvent.raiseEvent()
```

Comment:
Shouldn't this raise the `showPopupFragmentEvent` event?

If so, I would also rename the method to `showPopup()`. The doc should not mention anything about a Fragment.

Reply:
updated